### DISCUSSION TITLE
MooseFS ownership transition

### DISCUSSION THREAD

**Discussion by pkonopelko** [moosefs/moosefs#493] [General]
Dear MooseFS Users!
 
To maintain transparency of our internal processes that may be of interest to you, we would like to inform you that this year the ownership of and copyrights to the source code of MooseFS have been formally transferred from Core Technology sp. z o.o. to Tappest sp. z o.o., based on the formal agreement between the two companies. The latter has already been involved in MooseFS development and distribution for many years.

This change simplifies the structure of Intellectual Property ownership of the project. The transition will not affect the composition of the team. Our most experienced architects have been, they are, and they will continue to support the project.

We want to assure you that we will continue to make every effort for the development of MooseFS, as well as continue to look into the issues and discussions that you report – precisely as it has been happening so far.

At the same time, we would like to thank you so much for all your discussions, comments, and issues – they are very valuable and helpful. We are glad that thanks to you – our Community – MooseFS is getting better every day, and we appreciate that!

We value the importance of open source projects and despite the change of the owner, we plan to support Open Source MooseFS. Moreover, we work hard on MooseFS 4.x community release, hence you may have noticed less activity of our development team in the currently released 3.x branch lately. The Moose Is Coming, stay tuned!

Thank you,
MooseFS Team

**Comment by pkonopelko** [comment]
Dear MooseFS Users,

We would like to kindly inform you that as of January 1, 2023, as a result of a merger of Saglabs SA and Tappest sp. z o.o. through the acquisition of Tappest sp. z o.o. by Saglabs SA (incorporation), the entire enterprise of Tappest sp. z o.o., including all of its assets, rights and obligations, in particular – all of its rights, obligations and legal relationships related to the development and support of MooseFS and MooseFS Pro systems have been transferred to Saglabs SA by operation of law. The reason for this merger was to simplify the ownership structure of companies owned by the same shareholders.

In connection with the above, we would like to inform you that the entire team involved hitherto in MooseFS and MooseFS Pro systems development and support at Tappest sp. z o.o. will remain unchanged and continue to work in the same capacity and on the same terms within Saglabs SA. Thus, we will continue to provide the same high level of professional customer support for MooseFS and MooseFS Pro. In particular, we will continue to maintain the open-source version of MooseFS and develop both systems on an ongoing basis for the benefit of MooseFS Users.

Best regards,
MooseFS Team

**Reply by futzi** [reply]
Hi Piotr
is Saglabs still interessted in enhancing and supporting MooseFS?
I tried several times to get in touch via contact forms, mails etc, but now answers.
Probably here somebody is reading...

Best, Michael

**Reply by pkonopelko** [reply]
Hi Michael,
That’s strange! I’ll check this myself and get back to you today. Thank you very much for your message!
Best,
Piotr
